src/Features/Core/Portable/CodeFixes/Suppression/AbstractSuppressionCodeFixProvider.cs

```diff
@@ -195,7 +195,7 @@ internal class SuppressionTargetInfo
             var syntaxFacts = document.GetLanguageService<ISyntaxFactsService>();
 
             ISymbol targetSymbol = null;
-            var targetMemberNode = syntaxFacts.GetContainingMemberDeclaration(root, startToken.SpanStart);
+            var targetMemberNode = syntaxFacts.GetContainingMemberDeclaration(root, nodeWithTokens.SpanStart);
```

Comment:
For my education: in what case would these two values be different?  I would assume the start token of any node had the same `SpanStart` as that node.